docs/source/install.md

```diff
@@ -1,6 +1,6 @@
 # Install
 
-Featuretools is available for Python 3.7, 3.8, and 3.9. It can be installed from [pypi](https://pypi.org/project/featuretools/), [conda-forge](https://anaconda.org/conda-forge/featuretools), or from source.
+Featuretools is available for Python 3.7, 3.8, 3.9, and 3.10. It can be installed from [pypi](https://pypi.org/project/featuretools/), [conda-forge](https://anaconda.org/conda-forge/featuretools), or from source.
```

Comment:
```suggestion
Featuretools is available for Python 3.7, 3.8, 3.9, and 3.10. It can be installed from [pypi](https://pypi.org/project/featuretools/), [conda-forge](https://anaconda.org/conda-forge/featuretools), or from [source](https://github.com/alteryx/featuretools).
```

Reply:
done [f1aa760](https://github.com/alteryx/featuretools/pull/1940/commits/f1aa760d5730604cf65728488d03295cac627e09)